meta/store.go

```diff
@@ -394,6 +397,30 @@ func (s *Store) enableRemoteRaft() error {
 	return s.changeState(rr)
 }
 
+func (s *Store) checkRaftState() error {
+	peers, err := s.raftState.peers()
+	if err != nil {
+		return err
+	}
+
+	ni, err := s.Node(s.id)
+	if ni == nil {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+	if !contains(peers, ni.Host) {
+		return nil
+	}
+
+	if _, ok := s.raftState.(*localRaft); ok {
```

Comment:
I think you can remove this because `enableLocalRaft` does the same check at the start of the func.